src/Uno.UI/Microsoft/UI/Xaml/Controls/PullToRefresh/Native/RefreshContainer.iOS.cs

```diff
@@ -90,6 +90,7 @@ private void CleanupRefreshControl()
 			_refreshControl.EndRefreshing();
 			_refreshSubscription.Disposable = null;
 		}
+		_nativeScrollViewAttachment.Disposable = null;
```

Comment:
Is it UI testable?

Reply:
Not much unfortunately, as it is using the native refresh visualizer. But I can probably try to implement the test using screenshot testing - however I would prefer to merge this without the test now as Marc-Antoine is waiting for it

Reply:
Noted down in #9619